src/inmanta/ast/__init__.py

```diff
@@ -143,6 +143,38 @@ def __eq__(self, other: object) -> bool:
         return False
 
 
+class AnchorTarget(Location):
+    __slots__ = ("start_char", "end_lnr", "end_char", "docstring")
+
+    def __init__(
+        self,
+        file: str,
+        start_lnr: int,
+        start_char: Optional[int],
+        end_lnr: Optional[int],
+        end_char: Optional[int],
+        docstring: Optional[str],
+    ) -> None:
+        """
+        Create a new AncherTarget instance.
```

Comment:
I think there has to be an empty line here. Also a typo.
```suggestion
        Create a new AnchorTarget instance.

```